src/components/Listing/Listing.js

```diff
@@ -29,6 +29,14 @@ export default {
       url = url.replace('/{action}/{width}x{height}/', '/crop/420x236/')
```

Comment:
same replace logic. Consider to use a global function to normalize the URLs.

Reply:
Done it :)